.github/workflows/beta_build_and_deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Beta Build & Deploy Version
+
+on:
+  push:
+    tags:
+      - 'v*.*.*-beta'
+    branches-ignore:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: golang:1.12
+    steps:
+      - uses: actions/checkout@master
+      - name: dependencies
+        run: go mod download
+      - name: make test
+        run: make test
+      - name: beta release
+        env:
+          GITHUB_TOKEN: ${{ secrets.goreleaser }}
+        if: success()
+        run: |
+          go get github.com/goreleaser/goreleaser
+          goreleaser release --skip-validate --config=.beta-goreleaser.yml
```

Comment:
Specifying a newly created config file to handle the `beta` release via `goreleaser`